templates/zerver/base.html

```diff
@@ -14,8 +14,9 @@
                 {% endif %}
             {% endif %}
         {% endblock %}
-        <link id="favicon" rel="icon" href="{{ static('images/favicon.svg') }}?v=4" />
-        <link rel="alternate icon" href="{{ static('images/favicon.png') }}?v=4" />
+        <link id="favicon" rel="icon" href="{{ static('images/landing-page/hello/favicon.ico') }}" /> <!-- 32×32 -->
+        <link rel="alternate icon" href="{{ static('images/landing-page/hello/favicon.png') }}" />
+        <link rel="apple-touch-icon" href="{{ static('images/landing-page/hello/apple-touch-icon.png') }}"/> <!-- 180×180 -->
```

Comment:
So these change are to `base.html`, which gets included everywhere -- can you do them in a separate commit? I'm also not sure we want to have these icons be under `images/landing-page/hello/` -- since they are common things for all pages.

It's really really helpful to have the things that can be reviewed as "the /hello page looks great" separated from things that impact other pages, since 95% of the risk from this sort of PR is breaking something we didn't think to test.

Reply:
Done.

Reply:
Thanks, this helped reveal that this change has more implications than I'd imagined.

I think in any case, anything in `base.html` should not reference `images/landing-page` -- that's common configuration for every page in Zulip, so it should be in places like `images/logo/`.

Reply:
This changes the primary favicon from vector to bitmap, which seems like a strict regression.